riotfile.py

```diff
@@ -1,10 +1,13 @@
+import os
 from typing import List
 from typing import Tuple
 
 from riot import Venv
 from riot import latest
 
 
+os.environ["DD_TESTING_RAISE"] = "1"
```

Comment:
`riot` supports this too!

```python
venv = Venv(
    env={
        "DD_TESTING_RAISE": "1",
    }
)
```

Reply:
doing it this way will only work with `--pass-env` too, we always want this env variable to be set

Reply:
:+1 I seriously need to learn about all the riot features!